statement.md

```diff
@@ -0,0 +1,89 @@
+# JupyterLab Accessibility Statement
+
+## Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+
+## :construction: Draft Accessibility Statement for JupyterLab
```

Comment:
Thanks for the reminder here @isabela-pf - I always forget there are a million md subsets - replacing with the emoji
```suggestion
## 🚧 Draft Accessibility Statement for JupyterLab
```